packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -17,7 +17,7 @@ import {
 export type SortDirection = 'ascending' | 'descending' | 'none';
 
 export type SortState<T> = {
-  selection: keyof T;
+  selection: keyof T | undefined;
```

Comment:
Not sure i understand this change?

Reply:
It's so that I can set the upload table to have no default sorting so that when you try to re-add files, they appear at the top of the table

If we make it have a default sorting then it's not obvious that re-adding a file did anything because the sorting hides this update, if that makes sense

If a user does decide to sort the table and re-add an item then this UX issue is still there though

Reply:
Here's a video of what I'm talking about. I can remove these changes and just make the table have a default sort though, so no worries

In the video below, I'm re-adding the file `lighthouse-performance-results (2).zip` and can see it appear at the top of the table. Then I sort the column by name and try re-adding the same file, but it doesn't look like anything is happening because of the table sort making the file stay in it's sorted position:

https://github.com/user-attachments/assets/7b20dbca-6791-455f-967f-577ba1f9a4e1

Reply:
Let's continue sorting based on the end user selection, it's deterministic

Reply:
I think it's ok if it's not obvious that re-adding a file did anything. If the table is sorted, it should always go wherever the sort says it should go.

Reply:
okay thank you both! I'll undo these changes then and have the tasks table sort by `name` by default again